auth-web/src/App.vue

```diff
@@ -49,27 +49,32 @@ export default Vue.extend({
 
 </script>
 
-<style lang="stylus">
-  @import "./assets/styl/base.styl";
-  @import "./assets/styl/layout.styl";
-  @import "./assets/styl/overrides.styl";
+<style lang="scss">
+  @import "./assets/scss/base.scss";
```

Comment:
Is stylus to scss needed for this upgrade? Please add any reference so that we will have some reference in future.